pyintegration/deephaven2/constants.py

```diff
@@ -92,3 +93,56 @@ class SortDirection(Enum):
 MIN_POS_DOUBLE = _JQueryConstants.MIN_POS_DOUBLE
 """ Minimum positive value of type double. """
 
+
+class TimeZone(Enum):
```

Comment:
Should `TimeZone` go here, or in `time`?  Maybe `time`?

Reply:
agree